### PR TITLE
Add _posts/2021-11-24-#23512

### PR DESCRIPTION
**Pull request by MarcoFalke**

### CODE REVIEW

File: _posts/2021-11-24-#23512 [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+- While all nodes on the network are assumed to agree on consensus rules, they
+  might not agree on policy rules. For example, at times of high transaction
+  throughput, some peers might reject consensus-valid transactions with
+  insufficient fee.
+
+- Moreover, policy rules often change between software releases. For example if
+  a node receives a transaction that requires validation rules of a consensus
+  change it is not yet aware of, it may reject the loose transaction out of
+  caution even though the transaction will be valid if part of a block.
```
Comment (MarcoFalke) [inline _posts/2021-11-24-#23512:27]:
```suggestion
  caution even though the transaction will be valid in a block.
```

**Review by MarcoFalke — COMMENTED** [review]
netlify ping

File: _posts/2021-11-24-#23512 [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+   adverse effects. Hint: Is the Taproot activation set in stone? What happens
+   if a wallet creates a transaction with Taproot inputs before Taproot is
+   active?
+
+1. (bonus question) Is it theoretically possible for a mainnet chain to exist
+   that has Taproot not activated or activated at a different block height?
+
+1. (bonus question) Does this change affect miners running this code? Assume
+   that the miner is running on a chain that has Taproot not active. Would the
+   miner attempt to include the transaction in a block. Hint: Look at
```
Comment (MarcoFalke) [inline _posts/2021-11-24-#23512:62]:
```suggestion
   miner attempt to include the transaction in a block? Hint: Look at
```

**Review by MarcoFalke — COMMENTED** [review]
netlify ded?

**Comment by jnewbery** [comment]
Thanks @MarcoFalke!

**Pull request merged**